### ISSUE TITLE
测试demo时，train的问题

### ISSUE THREAD

**Issue by 619732457** [hukaixuan19970627/yolov5_obb#303]
python train.py --data 'data/yolov5obb_demo.yaml' --epochs 10 --batch-size 1 --img 1024 --device 0

/content/drive/MyDrive/colab/yolov5/yolov5_obb
train: weights=weights/yolov5m.pt, cfg=, data=data/yolov5obb_demo.yaml, hyp=data/hyps/obb/hyp.finetune_dota.yaml, epochs=10, batch_size=1, imgsz=1024, rect=False, resume=False, nosave=False, noval=False, noautoanchor=False, evolve=None, bucket=, cache=None, image_weights=False, device=0, multi_scale=False, single_cls=False, adam=False, sync_bn=False, workers=8, project=runs/train, name=exp, exist_ok=False, quad=False, linear_lr=False, label_smoothing=0.0, patience=100, freeze=[0], save_period=-1, local_rank=-1, entity=None, upload_dataset=False, bbox_interval=-1, artifact_alias=latest
github: up to date with https://github.com/hukaixuan19970627/yolov5_obb ✅
YOLOv5 🚀 78be0f1 torch 1.10.1+cu113 CUDA:0 (Tesla K80, 11441MiB)

hyperparameters: lr0=0.01, lrf=0.2, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=0.05, cls=0.5, cls_pw=1.0, theta=0.5, theta_pw=1.0, obj=1.0, obj_pw=1.0, iou_t=0.2, anchor_t=4.0, fl_gamma=0.0, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=180.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.5, fliplr=0.5, mosaic=0.95, mixup=0.1, copy_paste=0.0, cls_theta=180, csl_radius=2.0
Weights & Biases: run 'pip install wandb' to automatically track and visualize YOLOv5 🚀 runs (RECOMMENDED)
TensorBoard: Start with 'tensorboard --logdir runs/train', view at http://localhost:6006/
Downloading https://github.com/ultralytics/yolov5/releases/download/v6.1/yolov5m.pt to weights/yolov5m.pt...
100% 40.8M/40.8M [00:02<00:00, 19.5MB/s]

Overriding model.yaml nc=80 with nc=16

                 from  n    params  module                                  arguments                     
  0                -1  1      5280  models.common.Conv                      [3, 48, 6, 2, 2]              
  1                -1  1     41664  models.common.Conv                      [48, 96, 3, 2]                
  2                -1  2     65280  models.common.C3                        [96, 96, 2]                   
  3                -1  1    166272  models.common.Conv                      [96, 192, 3, 2]               
  4                -1  4    444672  models.common.C3                        [192, 192, 4]                 
  5                -1  1    664320  models.common.Conv                      [192, 384, 3, 2]              
  6                -1  6   2512896  models.common.C3                        [384, 384, 6]                 
  7                -1  1   2655744  models.common.Conv                      [384, 768, 3, 2]              
  8                -1  2   4134912  models.common.C3                        [768, 768, 2]                 
  9                -1  1   1476864  models.common.SPPF                      [768, 768, 5]                 
 10                -1  1    295680  models.common.Conv                      [768, 384, 1, 1]              
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 12           [-1, 6]  1         0  models.common.Concat                    [1]                           
 13                -1  2   1182720  models.common.C3                        [768, 384, 2, False]          
 14                -1  1     74112  models.common.Conv                      [384, 192, 1, 1]              
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 16           [-1, 4]  1         0  models.common.Concat                    [1]                           
 17                -1  2    296448  models.common.C3                        [384, 192, 2, False]          
 18                -1  1    332160  models.common.Conv                      [192, 192, 3, 2]              
 19          [-1, 14]  1         0  models.common.Concat                    [1]                           
 20                -1  2   1035264  models.common.C3                        [384, 384, 2, False]          
 21                -1  1   1327872  models.common.Conv                      [384, 384, 3, 2]              
 22          [-1, 10]  1         0  models.common.Concat                    [1]                           
 23                -1  2   4134912  models.common.C3                        [768, 768, 2, False]          
 24      [17, 20, 23]  1    812241  models.yolo.Detect                      [16, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [192, 384, 768]]
Model Summary: 369 layers, 21659313 parameters, 21659313 gradients, 50.6 GFLOPs

Transferred 475/481 items from weights/yolov5m.pt
Scaled weight_decay = 0.0005
optimizer: SGD with parameter groups 79 weight, 82 weight (no decay), 82 bias
albumentations: version 1.0.3 required by YOLOv5, but version 0.1.12 is currently installed
train: Scanning 'dataset/dataset_demo/labelTxt' images and labels...1 found, 0 missing, 0 empty, 0 corrupted: 100% 1/1 [00:00<00:00, 23.30it/s]
train: New cache created: dataset/dataset_demo/labelTxt.cache
val: Scanning 'dataset/dataset_demo/labelTxt.cache' images and labels... 1 found, 0 missing, 0 empty, 0 corrupted: 100% 1/1 [00:00<?, ?it/s]
Plotting labels to runs/train/exp/labels_xyls.jpg... 

AutoAnchor: 2.20 anchors/target, 0.982 Best Possible Recall (BPR). Current anchors are a good fit to dataset ✅
Image sizes 1024 train, 1024 val
Using 0 dataloader workers
Logging results to runs/train/exp
Starting training for 10 epochs...

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       0/9     1.15G    0.1215    0.1685   0.07387    0.2078        92      1024: 100% 1/1 [00:04<00:00,  4.03s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.27it/s]
                 all          1          0          0          0          0          0

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       1/9     2.05G    0.1243    0.1901   0.07483    0.2082       112      1024: 100% 1/1 [00:01<00:00,  1.29s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.29it/s]
                 all          1          0          0          0          0          0

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       2/9     2.05G      0.12    0.1834   0.07363    0.2059        77      1024: 100% 1/1 [00:01<00:00,  1.24s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.20it/s]
                 all          1          0          0          0          0          0

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       3/9     2.05G     0.115    0.1303   0.07681     0.206        53      1024: 100% 1/1 [00:01<00:00,  1.25s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.24it/s]
                 all          1          0          0          0          0          0

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       4/9     2.05G    0.1189    0.0965   0.07644    0.2094        18      1024: 100% 1/1 [00:01<00:00,  1.27s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.10it/s]
                 all          1          0          0          0          0          0

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       5/9     2.05G    0.1279    0.2276   0.07158    0.2085        93      1024: 100% 1/1 [00:01<00:00,  1.22s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.22it/s]
                 all          1          0          0          0          0          0

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       6/9     2.05G    0.1215    0.1586   0.07605    0.2046        65      1024: 100% 1/1 [00:01<00:00,  1.24s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.18it/s]
                 all          1          0          0          0          0          0

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       7/9     2.05G    0.1165    0.1214   0.07707     0.209        53      1024: 100% 1/1 [00:01<00:00,  1.24s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.33it/s]
                 all          1          0          0          0          0          0

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       8/9     2.05G    0.1201    0.1217   0.07699    0.2088        71      1024: 100% 1/1 [00:01<00:00,  1.22s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.55it/s]
                 all          1          0          0          0          0          0

     Epoch   gpu_mem       box       obj       cls     theta    labels  img_size
       9/9     2.05G     0.118    0.1409   0.07568     0.208        60      1024: 100% 1/1 [00:01<00:00,  1.22s/it]
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.59it/s]
                 all          1          0          0          0          0          0

10 epochs completed in 0.011 hours.
Optimizer stripped from runs/train/exp/weights/last.pt, 44.0MB
Optimizer stripped from runs/train/exp/weights/best.pt, 44.0MB

Validating runs/train/exp/weights/best.pt...
Fusing layers... 
Model Summary: 290 layers, 21640929 parameters, 0 gradients, 50.5 GFLOPs
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100% 1/1 [00:00<00:00,  3.45it/s]
                 all          1          0          0          0          0          0
Results saved to runs/train/exp

测试用的demo都是P0032，也没有显示未读取到相应的txt，运行时也没报错，但labels，P，R等数据的结果是0，请问这个问题可能出现在哪里？
